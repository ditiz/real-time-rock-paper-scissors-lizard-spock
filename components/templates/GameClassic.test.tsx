import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import GameClassicTemplate from "./GameClassic"

it("renders GameClassicTemplate unchanged", () => {
  const { container } = render(<GameClassicTemplate />)
  expect(container).toMatchSnapshot()
})
