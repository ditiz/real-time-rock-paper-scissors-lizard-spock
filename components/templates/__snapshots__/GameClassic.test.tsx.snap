// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GameClassicTemplate unchanged 1`] = `
<div>
  <nav
    class="navbar"
  >
    <div
      class="navbar__logo"
    >
      <img
        alt="logo"
        data-nimg="1"
        decoding="async"
        height="25"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
        style="color: transparent;"
        width="25"
      />
    </div>
    <div
      class="navbar__links"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/Rules"
      >
        Rules
      </a>
      <a
        href="/Classic"
      >
        Classic
      </a>
      <a
        href="/Advanced"
      >
        Advanced
      </a>
    </div>
  </nav>
  <header
    class="header"
  >
    <h1>
      Rock Paper Scissors
    </h1>
  </header>
  <main>
    <h2
      class="game__title"
    >
      Choose your move
    </h2>
    <article
      class="game"
    >
      <section>
        <img
          alt="rock icon"
          data-nimg="1"
          decoding="async"
          height="50"
          loading="lazy"
          src="icons/rock.svg"
          style="color: transparent;"
          width="50"
        />
      </section>
      <section>
        <img
          alt="paper icon"
          data-nimg="1"
          decoding="async"
          height="50"
          loading="lazy"
          src="icons/paper.svg"
          style="color: transparent;"
          width="50"
        />
      </section>
      <section>
        <img
          alt="scissors icon"
          data-nimg="1"
          decoding="async"
          height="50"
          loading="lazy"
          src="icons/scissors.svg"
          style="color: transparent;"
          width="50"
        />
      </section>
    </article>
  </main>
</div>
`;
